scripts/userstyles.yml

```diff
@@ -531,6 +533,14 @@ userstyles:
     readme:
       app-link: "https://www.linkedin.com"
     current-maintainers: [*isabelroses]
+  listenbrainz:
+    name: ListenBrainz
+    categories: [music, entertainment]
+    icon: musicbrainz
+    color: maroon
```

Comment:
Where is `maroon` from? We should go with the dominant colors on the website itself, which would either be `peach` or `mauve` in this case I think.

Reply:
I have a vague memory of copying this info from the MusicBrainz section in userstyles.yml, keeping the same colour for consistency between the different MetaBrainz sites.

But there _isn't_ a MusicBrainz section in userstyles.yml, so I actually have no idea where `maroon` came from. I'll swap it out for `peach`. 🤷‍♀️